submarine-sdk/pysubmarine/tests/models/test_model.py

```diff
@@ -54,7 +63,7 @@ def test_update_model(self, mocker):
     def test_load_model(self, mocker):
         mock_method = mocker.patch.object(mlflow.pyfunc, "load_model")
         mock_method.return_value = mlflow.pytorch._PyTorchWrapper(
-            LinearNNModel())
+            LinearNNModelTorch())
```

Comment:
Why only test pytorch?

Reply:
In this PR,I only deal with log_model function.By finitshed the expansion of this funtion,I think it will be easier to deal with other funtion(ex load_model) either in the same way or in Qlib's way.